cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraStorageSettings.java

```diff
@@ -66,6 +67,7 @@ public class CassandraStorageSettings extends CoreStorageSettings {
     private CassandraConsistencyLevel readConsistencyLevel = DEFAULT_CONSISTENCY_LEVEL;
     private String keyspace;
     private String schemaVersion = SCHEMA_VERSION;
+    private int randomAccessCacheSize = RANDOM_ACCESS_CACHE_SIZE;

```

Comment:
I think we need to add something like `Days` at the end. Because it is not clear what is the cache size refers to: number of pages or something else